src/tabs/pid_tuning.html

```diff
@@ -1377,6 +1377,32 @@
                                     </div>
                                 </td>
                             </tr>
+                            <tr class="newFilter dynamicNotch dynNotchCount">
+                                <td>
+                                    <input type="number" name="dynNotchCount" step="1" min="1" max="5"/>
+                                </td>
+                                <td>
+                                    <div>
+                                        <label>
+                                            <span i18n="pidTuningDynNotchCount"></span>
+                                        </label>
+                                        <div class="helpicon cf_tip" i18n_title="pidTuningDynNotchCountHelp" ></div>
+                                    </div>
+                                </td>
+                            </tr>
+                            <tr class="newFilter dynamicNotch dynNotchBandwidthHz">
+                                <td>
+                                    <input type="number" name="dynNotchBandwidthHz" step="1" min="1" max="1000"/>
```

Comment:
According to this new PR https://github.com/betaflight/betaflight/pull/10673 we would need to change `min="1"` to `min="6"` and `max="1000"` to `max="500"`